Resources/Prototypes/_NF/Entities/Mobs/NPCs/mob_hostile_base.yml

```diff
@@ -96,6 +109,30 @@
     #groups:
     #- AllAccess
 
+# Dungeon boss
+- type: entity
+  id: MobHostileBossBase
+  abstract: true
+  parent:
+  - MobEnhancedMovement
+  - MobPassiveRegenWeak
+  components:
+  - type: Hands
+  - type: Puller
+  - type: Stamina
+    critThreshold: 999
+  - type: MobPrice
+    price: 4000
+    deathPenalty: 0.5
```

Comment:
Isn't this a bit moot if the corpse destroys itself when leaving the expedition?

Reply:
Yeah. But maybe one day it'll be possible..